### PR TITLE
user guides: Review user documentation for Guides.

### PR DESCRIPTION
**Pull request by Ivche1337**
https://codein.withgoogle.com/dashboard/task-instances/6685400716279808/

### CODE REVIEW

File: user-guides/ivche1337/Setting-up-your-organization [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Setting up your organization
```
Comment (roberthoenig) [inline user-guides/ivche1337/Setting-up-your-organization]: You can make this a heading.

File: user-guides/ivche1337/Setting-up-your-organization [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+All the topics needed in order to setup a healthy zulip organisations are listed.
```
Comment (roberthoenig) [inline user-guides/ivche1337/Setting-up-your-organization]: Capitalize Zulip, and spell `organization` with a `z` (Zulip uses American English).

File: user-guides/ivche1337/Setting-up-your-organization [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+Every topic is well explained, and straight forward.
+
+All of the external topics needed in order to manage an organisation
+are listed in the tutorial, and those as well are well explained.
+
+Also, the screenshots provided in the guide, match my zulip development instance's..
```
Comment (roberthoenig) [inline user-guides/ivche1337/Setting-up-your-organization]: Are they two dots at the end of the phrase desired?

File: user-guides/ivche1337/getting-started-with-zulip.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+The documentation is descriptive,
+and it explains that zulip tries to achieve,
```
Comment (roberthoenig) [inline user-guides/ivche1337/getting-started-with-zulip.txt]: Capitalize Zulip.

File: user-guides/ivche1337/getting-started-with-zulip.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+and everything you need to start using zulip
+
+It isn't overwhelming, and the keybindings are well explained,
+and it is easy to follow through.
+
+I didn't have any issues regarding the tutorial,
+i followed through it, and everything was briefly explained.
```
Comment (roberthoenig) [inline user-guides/ivche1337/getting-started-with-zulip.txt]: Capitalize I

**Comment by Ivche1337** [comment]
Here, i fixed the things you pointed out, and also tidied up the commits :)
https://codein.withgoogle.com/dashboard/task-instances/6685400716279808/

**Comment by eeshangarg** [comment]
@Ivche1337: Thanks for working on this! Your comments look good, however you forgot to implement one piece of feedback @roberthoenig left! 

* The files should be renamed to end with `.md` since the task asks for Markdown files, not `.txt` files.
* Perhaps replace `(#Getting-started-with-Zulip)` with `# Getting started with Zulip` and `(#Setting-up-your-organization)` with `# Setting up your organisation`. Its cleaner that way and a conventional Markdown syntax!

The best way to make the above changes would be to rebase! Here's how to do that for each of the commits (you need to repeat what I say below for both commits):

1. Run `git rebase -i HEAD~2`
2. Replace `pick` with `edit` for the commit you want to change.
3. Make the changes and run `git add '*'`. Run `git status` and `git diff` to make sure you are on the right track.
4. Run `git commit --amend` and do not change the commit message. Your commit messages are great! :)
5. Run `git rebase --continue`.
6. Finally, `git push origin master --force`. And that should do it! :)

Let me know how it goes! Thanks! :)

**Comment by Ivche1337** [comment]
Done!

**Comment by eeshangarg** [comment]
@Ivche1337: Awesome! Merged! :)

**Pull request merged**